addons/purchase_mrp/tests/test_purchase_mrp_flow.py

```diff
@@ -1107,9 +1085,9 @@ def test_valuation_with_backorder(self):
         receipt.move_line_ids[0].quantity = 4
         receipt.move_line_ids[1].quantity = 2
         Form.from_action(self.env, receipt.button_validate()).save().process()
-        # Price Unit for 1 gm of the kit = 90000/1000 = 90
-        # unit_cost for cmp1 = 90 *1000* 3 / 2 / 2 / 1000 = 67.5
-        # unit_cost for cmp2  = 90 *1000* 3 / 2 / 1  * 1000 = 135000000
+        # Price Unit for 1 gm of the kit = 90000 / 1000 = 90
+        # unit_cost for cmp1 = 90 * 1000 * 3 / 2 / 2 / 1000 = 67.5
+        # unit_cost for cmp2  = 90 * 1000 * 3 / 2 / 1  * 1000 = 135000000
```

Comment:
like clesglow already said try to undo these changes. These linting diffs just make it harder to review/more confusing to read the code line history later on